### ISSUE TITLE
Black screen when clicking logo

### ISSUE THREAD

**Issue by sanders41** [meilisearch/documentation#806]
When I click on the MeiliSearch logo at the top left instead of zooming in on the logo I get a completely black screen. Looking at the [vuepress-plugin-zooming](https://github.com/vuepress/vuepress-plugin-zooming) github page the plugin has been archived and recommends using the [zooming](https://github.com/kingdido999/zooming) plugin now. I'm not sure if the vuepress-plugin-zooming has become outdated and maybe this is the cause for the issue? I checked Firefox, Chrome, and Safari and had the same issue in all 3 browsers.

**Comment by fharper** [comment]
My fault! I added the plugin last week, and didn't check the logo. I didn't know it was archived, it's the one that was recommended where I searched. Thanks a lot @sanders41, I'll do the switch!

**Comment by fharper** [comment]
Just checked, and it's archived because it was moved inside the vuepress-community repo: see https://github.com/vuepress/vuepress-community/tree/main/packages/vuepress-plugin-zooming. So it's still up to date as the vuepress plugin is using zooming in the back. As for the error with the logo, the issue should be created in the vuepress-community repo.

**Comment by fharper** [comment]
I'll let you create the issue unless you tell me you prefer we do it @sanders41  :)

**Comment by fharper** [comment]
Just realized, it may be because it's an image with a link...

**Comment by sanders41** [comment]
@fharper I don't mind submitting the bug report. I think I will check into the link idea first just to try to rule out anything like that before opening the report. Sound like a good plan?

**Comment by fharper** [comment]
Sounds good to me, thanks for being awesome @sanders41 !

**Comment by sanders41** [comment]
@fharper I played around in the browser inspector and figured out what the issue is. The image is doing the zooming, but because the navbar is a fixed position the zoom and position are in conflict with each other causing the black screen problem.

I tested a fix that makes the zoom only happen in the theme-default-content section, and that solved the problem. It does however mean the MeiliSearch logo in the navbar doesn't zoom any more. If that is OK have the change done and can open a PR.
